addons/stock/models/stock_orderpoint.py

```diff
@@ -240,6 +243,7 @@ def _compute_qty(self):
             for orderpoint in orderpoints_by_context:
                 orderpoint.qty_on_hand = products_qty[orderpoint.product_id.id]['qty_available']
                 orderpoint.qty_forecast = products_qty[orderpoint.product_id.id]['virtual_available'] + products_qty_in_progress[orderpoint.id]
+        self._compute_qty_to_order()
```

Comment:
not sure about the changes in this file, can you double check @amoyaux ?

Reply:
@sle-odoo  @amoyaux . Sorry, it's just for me to check if some issues are caused by the wrong qty_to_order. I'll remove that.